src/QuikGraph/Helpers/DisposableHelpers.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using JetBrains.Annotations;
+
+namespace QuikGraph.Utils
+{
+    /// <summary>
+    /// Helpers to work with <see cref="IDisposable"/>.
+    /// </summary>
+    public static class DisposableHelpers
```

Comment:
Note that I'm not really fan of making this stuff public since those are helpers for internal usage only, this is not the purpose of the library to provide such things imo.

Reply:
I had the idea to expose `DisposableHelpers` before I had the idea to give each observer a custom scope `struct`. My reasons for making it `public` are no longer relevant; would you like me to revert `DisposableHelpers`?